php/libraries/NDB_Page.class.inc

```diff
@@ -514,6 +554,26 @@ class NDB_Page
     }
 
     /**
+     * Creates an HTML radio button but does not add it to the form.
+     *
+     * @param string $field   The fieldname for this radio button
+     * @param string $label   The label to attach to this radio button
+     * @param string $options Options to pass to LorisForm
+     *
+     * @return QuickForm_Element representing radio button
+     */
+    function createRadio($field, $label, $options=null)
```

Comment:
Yeah that's exactly what I was thinking as well. These wrappers also exist for all of the other elements.